src/pages/CarsList/CarsList.tsx

```diff
@@ -55,13 +65,21 @@ function CarsList() {
     window.scrollTo(0, 0);
   }, [location]);
 
+  let fetchMoreData = () => {
+    setTimeout(() => {
+      if (carType == 3) dispatch(getMoreCars(""));
+      else dispatch(getMoreCars(carTypeList[carType]));
+    }, 2000);
```

Comment:
Non Actionable: I do not like this idea of you using a set timeout, but ok for now. I hope this is odne only to simulate loading/loaders.